src/models/users.js

```diff
@@ -1,11 +1,24 @@
 'use strict';
 module.exports = (sequelize, DataTypes) => {
-  const Users = sequelize.define('Users', {
-    first_name: DataTypes.STRING,
-    last_name: DataTypes.STRING
+  const users = sequelize.define('users', {
+    username: DataTypes.STRING,
+    firstName: DataTypes.STRING,
+    lastName: DataTypes.STRING,
+    email: DataTypes.STRING,
+    password: DataTypes.STRING,
+    isVerified: DataTypes.BOOLEAN,
+    roleId: DataTypes.INTEGER,
+    gender: DataTypes.STRING,
+    dateOfBirth: DataTypes.DATE,
+    preferredLanguage: DataTypes.STRING,
+    preferredCurrency: DataTypes.STRING,
+    residetialLocation: DataTypes.STRING,
+    department: DataTypes.STRING,
+    saveProfile: DataTypes.BOOLEAN,
```

Comment:
which property is saveProfile

Reply:
saveProfile is part of the field item the agreed on during the meeting we had together.